gatk-launch

```diff
@@ -153,20 +165,39 @@ def getSparkSubmitCommand(sparkSubmitCommand):
     else:
         return sparkSubmitCommand
 
-def getLocalGatkRunCommand():
+def getLocalGatkRunCommand(javaOptions):
     localJarFromEnv = getJarFromEnv(GATK_LOCAL_JAR_ENV_VARIABLE)
     if localJarFromEnv is not None:
+
+        # Add java options to our packaged local jar options
+        if javaOptions is not None:
+            PACKAGED_LOCAL_JAR_OPTIONS.extend(javaOptions.split(' '))
+
         return formatLocalJarCommand(localJarFromEnv)
 
     wrapperScript = getGatkWrapperScript(throwIfNotFound=False)
     if wrapperScript is not None:
+
+        # Add options to JAVA_OPTS environment var for dispatch script
+        if javaOptions is not None:
+            envJavaOpts = os.environ.get('JAVA_OPTS')
+            if envJavaOpts is not None:
+                envJavaOpts + ' ' + javaOptions
+            else:
+                envJavaOpts = javaOptions
+            os.environ['JAVA_OPTS'] = envJavaOpts
```

Comment:
Can you confirm that this successfully propagates the options to the wrapper script by adding in a temporary `echo` statement (or similar) to `build/install/gatk/bin/gatk` to print the full Java command line?